staging/src/k8s.io/cri-api/pkg/apis/runtime/v1/api.proto

```diff
@@ -330,6 +330,12 @@ message LinuxPodSandboxConfig {
     LinuxSandboxSecurityContext security_context = 2;
     // Sysctls holds linux sysctls config for the sandbox.
     map<string, string> sysctls = 3;
+
+    // overhead represents the overheads associated with this sandbox
+    LinuxContainerResources overhead = 4;
```

Comment:
how about "sandbox_overhead"?

Reply:
I'm a bit torn on that (but don't feel too strongly), since it is already part of the linux pod sandbox config. Sandbox's overhead, or sandbox's sandbox_overhead?

Reply:
just hope to keep symmetric with "sandbox_resources":) Or rename "sandbox_resources" as "resources"?